packages/block-library/src/button/edit.js

```diff
@@ -301,6 +317,7 @@ function ButtonEdit( props ) {
 						...colorProps.style,
 						...spacingProps.style,
 						...shadowProps.style,
+						...typographyProps.style,
```

Comment:
Hmm I think the `.wp-block-button__link` [height of 100%](https://github.com/WordPress/gutenberg/blob/main/packages/block-library/src/button/style.scss#L12-L12) is interfering with the typography setting  `writing-mode: vertical-rl;` 


<img width="243" alt="Screenshot 2024-12-17 at 11 07 42 am" src="https://github.com/user-attachments/assets/eadf61c6-7dcc-4755-8c4e-b58aed5ff4db" />

I wonder if changing it to `auto` would break anything else 🤔

Reply:
Good call! I didn't think of that, thanks!